mobile/src/bundles/common/components/slider/slider.tsx

```diff
@@ -0,0 +1,68 @@
+import CommunitySlider from '@react-native-community/slider';
+import React from 'react';
+import { type StyleProp, type ViewStyle } from 'react-native';
+
+import { Pressable, Text, View } from '~/bundles/common/components/components';
+import { Color, TextCategory } from '~/bundles/common/enums/enums';
+import { useWindowDimensions } from '~/bundles/common/hooks/hooks';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+import { styles } from './styles';
+
+type SliderProperties = {
+    thumbTitleValue: string;
+    sliderValue: number;
+    onSliderValueChange: (value: number) => void;
+    onResetClickHandler: () => void;
+    containerStyle?: StyleProp<ViewStyle>;
+    minSliderValue?: number;
+    maxSliderValue?: number;
+    thumbTitleValueWidth?: number;
+    resetTitle?: string;
```

Comment:
This should be just initial value (min value), for example 0 which also means no

Reply:
Didn't understand it, we need min value for slider width, active value, and set value we got from parent component and return it to actual for form controller.
Use thumbTitleValueWidth for calculation offset and set it to text component. Maybe we should use a better way for it?)
Noted that I should remove reset title and so on, because this component didn't need to know about them)

Reply:
My comment was about resetTitle, GitHub displayed it wrong

Reply:
fix